tt_metal/llrt/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# add CFLAG  -Wno-int-to-pointer-cast
+
+set(LLRT_SRC
+    ${CMAKE_CURRENT_SOURCE_DIR}/llrt.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/rtoptions.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tt_cluster.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tt_hexfile.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tt_memory.cpp)
+
+add_library(llrt OBJECT ${LLRT_SRC})
```

Comment:
changed now so that tensor, queue, tt_dnn now builds into objects and those objects are used to build `tt_eager.so` , and then that .so is used for the eager cpp tests